README.md

```diff
@@ -35,7 +35,7 @@ to get it working - please see below for how to get support.
 *   FFmpeg (if you are on Ubuntu or Debian, see first see
     [issue 32](https://github.com/aspiers/ly2video/issues/32))
 *   TiMidity++
-*   Python 2.7 (please note Python 3 is not yet supported!)
+*   Python 3.5
```

Comment:
What about newer versions?

Reply:
I've been testing with Python 3.5 (Ubuntu 16.04) and Python 3.9 (Debian Bullseye), so I there shouldn't be an issue with newer versions.
Would specifying Python 3.5+ be enough?

We'll need to update the minimum supported version if we upgrade [Pillow](https://pillow.readthedocs.io/en/stable/installation.html#python-support) or use any new Python feature.

Reply:
That's great, thanks.